src/scss/objects/_form-toggle.scss

```diff
@@ -164,3 +182,8 @@ $formToggle-button-background       : $blue--offWhite;
         right: spacing();
         transform: translateY(-50%);
     }
+
+    .o-formToggle--disabled {
```

Comment:
Is this class just used to add disabled styling to the parent?

Reply:
that's correct :)